### DISCUSSION TITLE
ICU tokenizer config

### DISCUSSION THREAD

**Discussion by Negatifff** [osm-search/Nominatim#2796] [Q&A]
Hello

Switched from version 3 to version 4 and of course found a new tokenizer icu.
I read the official documentation https://nominatim.org/release-docs/latest/customize/Tokenizers/#icu-tokenizer, but based on the example given in the documentation (especially when the example contains nested files with unknown content) is a rather non-trivial task . In addition, with the inability to see the changes made immediately without the time-consuming import of OSM data.

Maybe someone already has a ready-made config for the icu tokenizer? Or maybe there is somewhere (or someone) a config for the icu tokenizer, which repeats the settings of the old legacy tokenizer? it would be great.

**Comment by lonvia** [comment]
The ICU tokenizer comes with a [standard configuration](https://github.com/osm-search/Nominatim/blob/master/settings/icu_tokenizer.yaml). This is used if you don't set up anything special. Also, since 4.1, it is the default configuration for Nominatim.

**Comment by Negatifff** [comment]
yes, I found it. thanks
